TFUtil.py

```diff
@@ -8638,11 +8638,11 @@ def get_positional_encoding(num_channels, length=None, position=None, min_timesc
 
   :param int num_channels: scalar, size of timing embeddings to create. The number of
     different timescales is equal to channels / 2.
-  :param tf.Tensor|None length: scalar, length of timing signal sequence. if not given, is shape(position)[0]
-  :param tf.Tensor|None position: could be provided directly. int32, shape (length,)
-  :param float min_timescale: a float
-  :param float max_timescale: a float
-  :return: a Tensor of timing signals (length, channels)
+  :param tf.Tensor|None length: scalar, length of timing signal sequence.
+  :param tf.Tensor|None position: could be provided directly. int32, shape (length,) or (batch, length).
```

Comment:
It does not really matter at all what the axes are, does it? Then you can just write "can have any shape". It's a bit confusing that you write it must be "(batch,length)", while it does not need to be that way, and I think even you yourself also use it differently.

Reply:
Done.